server/server.js

```diff
@@ -1,14 +1,20 @@
 const express = require('express');
-const app = express();
+const app = express();  // ?? This is basically us starting up the server
 const port = 3000;
 const path = require('path');
 
-app.use(express.static('./dist'))
```

Comment:
We need this to serve static assets. Does this interfere with your code? If not, let's preserve this line.